ui/core/components/multi_icon_picker.ts

```diff
@@ -88,10 +89,18 @@ export class MultiIconPicker<ModObject> extends Component {
 
 		if (this.currentValue) {
 			this.dropdownRootElem.classList.add('active');
-			this.currentValue.fillAndSet(this.buttonElem, false, true);
+			if (this.config.categoryId != null) {
```

Comment:
Instead of changing it like this, you could revert all changes to this function and instead modify the `getMaxValue` function below. Something like ```
const actionId = this.pickers.map(...);
if (this.categoryId == null) { return actionId; } else { return actionId == null ? null : categoryId; }

Reply:
That would make the category icon get the `active` class and show that a buff is active in this category without any being active (it was how i did it the first time)